pnpm-lock.yaml

```diff
@@ -360,6 +360,19 @@ packages:
     resolution: {integrity: sha512-0hYQ8SB4Db5zvZB4axdMHGwEaQjkZzFjQiN9LVYvIFB2nSUHW9tYpxWriPrWDASIxiaXax83REcLxuSdnGPZtw==}
     dev: true
 
+  /@colors/colors@1.6.0:
```

Comment:
Where did these changes come from? I don't see any package.json changes and I don't get changes when I do `pnpm install`

Reply:
Weird, I also don't get changes when doing an install on this branch. No idea where the packages are coming from, but I guess its fine